docs/guides/about/glossary.md

```diff
@@ -353,6 +356,11 @@ a typical desktop computer or workstation in order to solve large problems in sc
 
 The angle between a ray incident on a surface and the line perpendicular to the surface at the point of incidence.
 
+{#ingesting}
+## Ingesting
```

Comment:
Let's drop this one - we no longer do ingestion in DEA and newer versions of datacube don't support it at all